From 8e9afb5438a592e4142ed4878eeca152b8782d09 Mon Sep 17 00:00:00 2001
From: "winton.liu" <admin@brahmaos.io>
Date: Mon, 3 Sep 2018 15:15:46 +0800
Subject: [PATCH] build: update img zip file name

1. Update img zip filename to 'aosp_angler-img-userdebug-<BrahmaosVersion>-<date>.zip'
2. Image zip delete userdata.img

Change-Id: Iec1228c7d6ad686978beb9077665799fcd0c844e
Signed-off-by: winton.liu <admin@brahmaos.io>
---
 core/Makefile            | 6 ++++--
 core/version_defaults.mk | 2 +-
 2 files changed, 5 insertions(+), 3 deletions(-)

diff --git a/core/Makefile b/core/Makefile
index 618e983..3fc3330 100644
--- a/core/Makefile
+++ b/core/Makefile
@@ -2636,14 +2636,16 @@ name := $(TARGET_PRODUCT)
 ifeq ($(TARGET_BUILD_TYPE),debug)
   name := $(name)_debug
 endif
-name := $(name)-img-$(FILE_NAME_TAG)
+name := $(name)-img-$(FILE_NAME_TAG)-$(BUILD_ID)-$(shell $(DATE) +%Y%m%d)
 
 INTERNAL_UPDATE_PACKAGE_TARGET := $(PRODUCT_OUT)/$(name).zip
 
 $(INTERNAL_UPDATE_PACKAGE_TARGET): $(BUILT_TARGET_FILES_PACKAGE) $(ZIP2ZIP)
 	@echo "Package: $@"
+	@echo "$(ZIP2ZIP) -i $(BUILT_TARGET_FILES_PACKAGE) -o $@"
 	$(hide) $(ZIP2ZIP) -i $(BUILT_TARGET_FILES_PACKAGE) -o $@ \
-	   OTA/android-info.txt:android-info.txt "IMAGES/*.img:."
+	   OTA/android-info.txt:android-info.txt IMAGES/vendor.img:vendor.img \
+	   IMAGES/system.img:system.img  IMAGES/boot.img:boot.img  IMAGES/recovery.img:recovery.img
 
 .PHONY: updatepackage
 updatepackage: $(INTERNAL_UPDATE_PACKAGE_TARGET)
diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index 648a21d..02ca1d0 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -223,5 +223,5 @@ ifndef BUILD_NUMBER
   # If no BUILD_NUMBER is set, create a useful "I am an engineering build
   # from this date/time" value.  Make it start with a non-digit so that
   # anyone trying to parse it as an integer will probably get "0".
-  BUILD_NUMBER := eng.$(shell echo $${USER:0:6}).$(shell $(DATE) +%Y%m%d.%H%M%S)
+  BUILD_NUMBER := $(TARGET_BUILD_VARIANT)
 endif
-- 
2.7.4

